frontend/src/pages/Matching.vue

```diff
@@ -219,23 +222,23 @@
               <div class="subheader-right">
                 <button
                   class="button field is-white"
-                  @click="checkedRows = []"
-                  :disabled="!checkedRows.length"
+                  @click="unmatchSelected()"
+                  :disabled="!selectedMatches.length"
                 >
-                  <span>Unmatch Selected </span> ({{ checkedRows.length }})
+                  <span>Unmatch Selected </span> ({{ selectedMatches.length }})
                 </button>
                 <div class="divider"/>
                 <button
                   class="button field is-white"
-                  @click="cardModal()"
-                  :disabled="!checkedRows.length"
+                  @click="confirmSelected()"
```

Comment:
If we use the same function `confirmSelected` for both "Confirm Selected" and "Confirm All", we will need a way to differentiate when each button is clicked. Consider:

- "Confirm All" means confirm all the rows in `matchedPairs`, regardless of whether they are selected or not;
- "Confirm Selected" means confirm the rows in `matchedPairs` which are also selected.

So we need a way to tell the `confirmSelected` function whether or not to care about the selected status.

Reply:
Added `confirmAll()` method.